Extractor/Pushers/FDM/Types/ReferenceTypes.cs

```diff
@@ -0,0 +1,44 @@
+using Cognite.OpcUa.Nodes;
+using Cognite.OpcUa.Types;
+using CogniteSdk.Beta.DataModels;
+using Opc.Ua;
+
+namespace Cognite.OpcUa.Pushers.FDM.Types
+{
+    public class ReferenceNodeType : ReferenceNode
```

Comment:
So this renaming doesn't really make sense. A "Reference Node Type" is, presumably, the type of a "Reference Node", whatever that is, which is not what this is. I used "NodeTypeReference" because it is a reference from a Node Type.